<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>225</red>
        <green>255</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>240</red>
        <green>240</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>225</red>
        <green>255</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Super Turbo DNA Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="ScoreSet_widget" native="true">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>120</width>
      <height>141</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <widget class="QLabel" name="ScoreSet_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Score Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="Match_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QLabel" name="Match_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Match</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Match_spinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Mismatch_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QLabel" name="Mismatch_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Mismatch</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Mismatch_spinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>-99</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Gap_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <widget class="QLabel" name="Gap_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Gap</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="Gap_spinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>0</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="minimum">
       <number>-99</number>
      </property>
      <property name="value">
       <number>-2</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="History_widget" native="true">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>0</y>
      <width>220</width>
      <height>530</height>
     </rect>
    </property>
    <widget class="QLabel" name="History_label">
     <property name="geometry">
      <rect>
       <x>-2</x>
       <y>0</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>History</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QListWidget" name="History_listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>220</width>
       <height>500</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QListWidget {
		background-color: Gainsboro;
}

QListWidget::item {
		background-color: white;
		/*color black;*/
        margin-bottom: 1px;
		padding: 0px;
}

QListWidget::item::label {
		color red;
}</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="uniformItemSizes">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="DataInputLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="Seq1_widget" native="true">
       <widget class="QLabel" name="Seq1Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>431</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sequence 1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="Seq1TextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>431</width>
          <height>391</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Input sequence here in FASTA format or load from file using the button below</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Seq1loadButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>480</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load from FASTA file</string>
        </property>
       </widget>
       <widget class="QLabel" name="Seq1GCLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Seq2_widget" native="true">
       <widget class="QLabel" name="Seq2Label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>431</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Sequence 2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="Seq2TextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>431</width>
          <height>391</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="plainText">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Input sequence here in FASTA format or load from file using the button below</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Seq2loadButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>480</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Load from FASTA file</string>
        </property>
       </widget>
       <widget class="QLabel" name="Seq2GCLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="ProcessedDataWidget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>529</y>
      <width>1041</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QTextBrowser" name="alignmentBrowser">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>698</width>
       <height>111</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier</family>
      </font>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="processButton">
     <property name="geometry">
      <rect>
       <x>359</x>
       <y>0</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Process Data</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="loadHistoryButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>540</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
